test/dotcom/lib/redirector_test.exs

```diff
@@ -18,23 +18,50 @@ defmodule DotcomWeb.RedirectorTest do
   end
 
   test "route redirected to internal route", %{conn: conn} do
-    conn = Redirector.call(conn, to: "/miami")
+    conn = call(conn, to: "/miami")
```

Comment:
Is there something special about Miami? If not, this should be a random string. Same with all others below.

Reply:
Agreed - let's do that as a follow-up.

Reply:
Why a follow-up? This has a week before it needs to be launched, right?

Reply:
https://www.notion.so/mbta-downtown-crossing/The-Four-Legs-of-Quality-Programming-b35248799da84e02b93887361d5bf343